pkg/tracker/currentVariables_test.go

```diff
@@ -29,8 +30,23 @@ func TestCurrentVariables(t *testing.T) {
 
 	tracker := &CurrentVariablesTracker{}
 
+	ctx := context.WithValue(context.Background(), tellorCommon.ClientContextKey, client)
```

Comment:
This should be all done in the `testutil.CreateContext(t)`, no? If it isn't best to add it there.